src/test/java/com/bestcommerce/customer/integration/controller/CustomerControllerTest.java

```diff
@@ -101,4 +101,19 @@ public void updateCustomerTest() throws Exception{
         assertThat(dto.getCustomerBirthDate()).isEqualTo(modifyCustomer.getBirthdate());
         assertThat(LocalDate.now().toString()).isEqualTo(modifyCustomer.getModifyDate());
     }
+
+    @DisplayName("회원 정보 조회")
+    @Test
+    public void getCustomerTest() throws Exception{
+
+        String customerEmail = "dudtkd0219@gmail.com";
+        CustomerDto dto = new CustomerDto(0L,"",customerEmail,"","","","","");
+
+        String content = objectMapper.writeValueAsString(dto);
+
+        mockMvc.perform(post("/account/get").contentType(MediaType.APPLICATION_JSON).content(content))
+                .andExpect(status().isOk())
+                .andDo(MockMvcResultHandlers.print());
+
+    }
```

Comment:
entity id is very important value.
so now on our project, just give email value to know their information